library/errata_tool_product_version.py

```diff
@@ -281,6 +295,7 @@ def run_module():
         brew_tags=dict(type='list', required=True),
         use_quay_for_containers=dict(type='bool'),
         use_quay_for_containers_stage=dict(type='bool'),
+        suppress_push_request_jira=dict(type='bool', default=None),
```

Comment:
Ansible already sets `default=None` for all parameters, so you can remove it here.

I had to learn this the hard way, and I went through the codebase a while back and purged all the `default=None` instances in 847dc6110f1b7817aed0c2a04c6d8eb032f2b8b2 and 654df3784d83f7dadc15fbac4057299a7bae4e98

Reply:
Thanks for that insight, removed